guacamole/src/main/webapp/translations/de.json

```diff
@@ -138,7 +153,7 @@
 
     "FORM" : {
 
-        "FIELD_PLACEHOLDER_DATE" : "YYYY-MM-DD",
```

Comment:
`FIELD_PLACEHOLDER_DATE` shows the format of date required by the application, which is always YYYY-MM-DD (ISO 8601). If the letters, etc. used within the date format need to be different for the language at hand, then it should be updated accordingly, but the format does still need to equate to YEAR-MONTH-DAY.

I'm guessing that the German version of the required format would be "JJJJ-MM-TT", but I don't honestly know.